components/molecules/RepoRow/repo-row.tsx

```diff
@@ -172,7 +175,7 @@ const RepoRow = ({ repo, topic, userPage, selected, handleOnSelectRepo }: RepoPr
             <div>Pr Velocity</div>
             <div className="flex text-base gap-x-3">
               <div>{prVelocityInDays}</div>
-              { repo.id ? <Pill text={`${prsMergedPercentage}%`} size="small" color="green" /> : "" }
+              {repo.id ? <Pill color="purple" text={`${getPrsPercent(totalPrs, prVelocityCount || 0)}%`} /> : ""}
```

Comment:
```suggestion
              {repo.id ? <Pill color="purple" text={`${getPrsPercent(totalPrs, prsMergedPercentage || 0)}%`} /> : ""}
```

PR velocity is not providing what we need.  Instead use the amount of PRs merged in the filtered time.